modules/services.sh

```diff
@@ -97,9 +97,12 @@ function services {
               check_tor "8080"
               ;;
             nextcloud)
-              docker run --name nextcloud -d -p 8081:80 nextcloud
-              echo "nextcloud built and started"
-              check_tor "8081"
+              bash $TEMPLATES/services/nextcloud/nextcloud_yml.sh
+              echo "yml file created"
+
+              docker-compose -f /srv/nextcloud/nextcloud.yml -p nextcloud up -d
+              echo "Next cloud built and started"
+              check_tor "8081:80"
```

Comment:
When you `check_tor`, you should be checking for the specific port of `8081`, not `8081:80`.

If you take a look at the `check_tor` function, you can see that we are just greping the output of `treehouses tor list`. You can run `treehouses tor list` to see what the output is like to know what to pass to the `check_tor` function.